backend/copr_backend/background_worker_build.py

```diff
@@ -381,6 +381,18 @@ def _proctitle(self, text):
         self.log.debug("setting title: %s", text)
         self.setproctitle(text)
 
+    def _check_build_interrupted(self):
+        """
+        Should we interrupet a running worker?
+        """
+        # Was the build canceled by the user?
+        if self._cancel_task_check_request():
+            return True
+
+        # Is the resalloc ticket healthy?
+        self.host.ticket.collect()
```

Comment:
The cancel check period is 5 seconds .... I'd propose doing this at most every minute, maybe 5 minutes.
Just to not cause unnecessary `backend <-> resalloc` traffic.

Reply:
The `collect()` may hang when the server goes down because we are using `survive_server_restart`: https://github.com/praiskup/resalloc/blob/7a46fcc269a9e38174786ca3fb365e96c0a46cbd/resalloc/client.py#L43-L53

Might be OK, ... but then, another potential problem is that not all the exceptions are being caught.

Reply:
I will kill resalloc server and see what happens.